exercises/space-age/.meta/template.j2

```diff
@@ -0,0 +1,23 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header(["SpaceAge"]) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {%- set seconds = case["input"]["seconds"] %}
+        {%- set planet = case["input"]["planet"] %}
+        self.assertEqual(SpaceAge({{ seconds }}).on_{{ planet | lower }}(), {{ case["expected"] }})
+
+    {% endfor %}
+    {% if additional_cases | length -%}
+
+    # Additional tests for this track
+
+    {% for case in additional_cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {%- set seconds = case["input"]["seconds"] %}
+        self.assertEqual(SpaceAge({{ seconds }}).seconds, {{  case["expected"]}})
```

Comment:
```suggestion
        self.assertEqual({{ case["property"] }}({{ seconds }}).seconds, {{  case["expected"]}})
```

What if we made this change instead? 

Is it more important for `additional_tests.json` to be consistent with the format of `canonical-data.json`? If not, this change could make the template more flexible to adding future "additional tests".

Reply:
I would say the above change actually makes the template *less* flexible to adding future additional tests.

The more I look at this, I wonder why this additional test is even present. I'm becoming an advocate for removing it entirely.

@yawpitch, what do you think of that?

Reply:
I see no reason why this additional test exists at all, as it's just testing that the input is conserved on the instance ... as I also see no reason to implement this as a class in the first place I'm not sure why we'd bother testing that.

Reply:
The class-based implementation is probably non-essential, but changes to that structure should probably wait for a separate PR.

@crsmi I think we should just remove the additional test altogether. This will simplify the template, remove some noise from the test file, and should clear up any more disputes with this PR.

Reply:
Alright, I removed the additional test.